libs/category/guides/architecture/filtering/idempotence.md

```diff
@@ -0,0 +1,7 @@
+# Idempotence
+Daffodil applies filters "idempotently". That is to say, when you apply a filter operation multiple items, you get the same end result. 
```

Comment:
*multiple times

Reply:
use of 2nd person should be removed

Reply:
8b2f50a1

Reply:
use of 2nd person should be removed

Reply:
49a89da757bf859c58dbdc95ee2df0a85827834c